docarray/document/mixins/_property.py

```diff
@@ -1,12 +1,12 @@
-# auto-generated from /Users/hanxiao/Documents/docarray/scripts/gen_doc_property_mixin.py
-from typing import TYPE_CHECKING, Dict, List, Optional
+# auto-generated from /Users/marco/code/opensource/docarray/scripts/gen_doc_property_mixin.py
```

Comment:
remove this auto generated line

Reply:
I suggest to keep the comment, because it tells that `_property.py` is auto-generated using `scripts/gen_doc_property_mixin.py`. I can change `scripts/gen_doc_property_mixin.py`, so that the comment will look like `# auto-generated from docarray/scripts/gen_doc_property_mixin.py`. Is that OK?

Reply:
okey is good